README.md

```diff
@@ -2,35 +2,41 @@
 
 ## General information
 
-VSCode stores its configuration in JSON files. You can store the configuration either in your user directory or in your workspace, the current project you work.
+VSCode stores its configuration in JSON files. Depending on the scope your
+settings apply to, you can store the configuration either in your user directory
+or in your workspace, the current project you work.
 
-User configuration files are stored in one of the following directories:
+User configuration files apply settings to all project that you open in VScode.
+They are stored in one of the following directories:
 
 * `~/.config/Code/User/` (VSCode)
 * `~/.config/VSCodium/User/` (VSCodium)
 
-Inside your workspace (the project directory), configuration can be stored in a `.vscode` subdirectory.
-
-I prefer to use the per-directory approach which is useful if you
-have checkouts/projects of the same kind in a single directory on the file
-system.
+Workspace configuration files apply settings to the current workspace
+(directory) only and have precedence over user settings. They are stored in the
+`.vscode` subdirectory of the project.
```

Comment:
I have some suggestions:

* Technically, it's probably correct if you write about "Workspace configuration files". However, from a user's perspective, it doesn't really matter. What's important here is that workspace project configs take precedence over user config. That's all what they need IMHO.
* What's "They" in the second sentence?
* The parenthesis with directory is a bit distracting and confusing for my taste. I'd like to avoid it.

If I take all into account, I'd suggest:

```suggestion
Settings in the current project's workspace take precedence over user settings.
The project's workspace settings are stored in the `.vscode` subdirectory.
```